doc/neps/nep-0000.rst

```diff
@@ -0,0 +1,219 @@
+=======================
+NEP Purpose and Process
+=======================
+
+:Author: Jarrod Millman <millman@berkeley.edu>
+:Status: Draft
+:Type: Process
+:Created: 2017-12-11
+
+
+What is a NEP?
+--------------
+
+NEP stands for NumPy Enhancement Proposal.  A NEP is a design
+document providing information to the NumPy community, or describing
+a new feature for NumPy or its processes or environment.  The NEP
+should provide a concise technical specification of the feature and a
+rationale for the feature.
+
+We intend NEPs to be the primary mechanisms for proposing major new
+features, for collecting community input on an issue, and for
+documenting the design decisions that have gone into NumPy.  The NEP
+author is responsible for building consensus within the community and
+documenting dissenting opinions.
+
+Because the NEPs are maintained as text files in a versioned
+repository, their revision history is the historical record of the
+feature proposal [1]_.
+
+
+Types
+^^^^^
+
+There are three kinds of NEP:
+
+1. A **Standards Track** NEP describes a new feature or implementation
+   for NumPy.
+
+2. An **Informational** NEP describes a NumPy design issue, or
```

Comment:
I wonder about the benefit of this type of NEP; can you think of an example where it would be used?  The description states that it does *not* represent the NumPy community consensus, but since that is our governance agreement, it's unclear to me how these NEPs would fit in.

Reply:
I wondered the same thing, so I just removed it.